tests/test_bug_features.py

```diff
@@ -117,3 +122,5 @@ def test_comment_count():
 
 def test_comment_length():
     read('comment_length.json', comment_length, [566, 5291])
+
```

Comment:
Now you have two empty lines here :rofl:

Reply:
I feel like an idiot :sob:

Reply:
No worries, it happens to everyone :P